### DISCUSSION TITLE
Multiple nodes on one phone?

### DISCUSSION THREAD

**Discussion by ronnietucker** [markqvist/Reticulum#645] [Sideband]
Hi folks,
Just set up my first node (Heltec V3) with my phone. I have another Heltec, but how do I set it up with my phone seemingly linked to Heltec1? 

With Meshtastic I could setup multiple nodes with my single phone and switch between them. Is this possible with Sideband?

**Comment by markqvist** [comment]
If you want to use multiple different RNodes from the same instance of Sideband, you can do that simply by having them all paired to the same device over Buetooth (or simply connecting the one you want to use at any given moment via USB).

If you want to use multiple different RNodes as separate interfaces *at the same time*, this is currently only possible by using the advanced configuration options, and manually specifying interface configurations for it - there is no GUI options for handling this at the moment.
